app/models/form/sales/questions/buyer1_income_known.rb

```diff
@@ -0,0 +1,32 @@
+class Form::Sales::Questions::Buyer1IncomeKnown < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "income1nk"
+    @check_answer_label = "Buyer 1’s gross annual income"
+    @header = "Do you know buyer 1’s annual income?"
+    @type = "radio"
+    @answer_options = ANSWER_OPTIONS
+    @page = page
+    @hint_text = "What counts as income?
```

Comment:
I'm not sure a multiline string is going to translate onto the page. Could you double check?

Reply:
@jamesrose this is apparently supposed to be a details component, I left a comment for you here:

https://digital.dclg.gov.uk/jira/browse/CLDC-1512

What would you recommend going for?